src/components/login-form.js

```diff
@@ -1,14 +1,110 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteForm from './vote-form';
 
 export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
+	const [logForm, setLogForm] = useState({ email: '', password: '' });
+	const [isloged, setloged] = useState(false);
+
+	const erasePW = () => {
+		document.getElementById('password').value = '';
+	};
+
+	const eraseEmail = () => {
+		document.getElementById('email').value = '';
+	};
+
+	const handleFormChange = (e) => {
+		setLogForm({ ...logForm, [e.target.id]: e.target.value });
+	};
+
+	const handleSubmit = (logForm) => {
+		const { email, password } = logForm;
```

Comment:
```suggestion
	const handleSubmit = () => {
		const { email, password } = loginForm;
```
handleSubmit 함수내에서 loginForm에 바로 접근할 수 있기 때문에 parameter로 넘겨주지 않아도 될 것 같아요!